internal/provider/models/engine.go

```diff
@@ -305,6 +321,43 @@ func ConditionGroupsAttribute() schema.SetNestedAttribute {
 	}
 }
 
+func GroupingKeysAttribute() schema.SetNestedAttribute {
+	return schema.SetNestedAttribute{
+		MarkdownDescription: "The attributes to group alerts by",
+		Required:            true,
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: map[string]schema.Attribute{
+				"id": schema.StringAttribute{
+					MarkdownDescription: "The unique identifier of the alert attribute to group on",
+					Required:            true,
+				},
+				"reference": schema.SingleNestedAttribute{
```

Comment:
Should this be in the tf config? I don't really understand this all that well but I thought this was just returned from the API as a presentational thing?